matchbox_socket/src/ggrs_socket.rs

```diff
@@ -49,3 +62,11 @@ impl ggrs::NonBlockingSocket<PeerId> for WebRtcSocket {
         messages
     }
 }
+
+impl WebRtcSocketBuilder {
+    /// Adds a new channel configured correctly for usage with GGRS to the [`WebRtcSocket`].
+    pub fn add_ggrs_channel(mut self) -> Self {
```

Comment:
Shouldn't this be a feature method?

Reply:
It is, because it's in the ggrs_socket module, which is conditionally included.

Reply:
D'oh. That's right.